cg/store/api/findbusinessdata.py

```diff
@@ -132,6 +132,18 @@ def family_samples(self, family_id: str) -> List[models.FamilySample]:
             .all()
         )
 
+    def get_samples_by_family_id(self, family_id: str) -> List[models.Sample]:
+        """Get samples on a given family_id"""
+
+        case: models.Family = self.family(internal_id=family_id)
+        return [link.sample for link in case.links] if case else []
+
+    def get_sequenced_samples(self, family_id: str) -> List[models.Sample]:
+        """Get sequenced samples by family_id"""
+
+        samples: List[models.Sample] = self.get_samples_by_family_id(family_id)
+        return [sample for sample in samples if sample.sequenced_at]
```

Comment:
The logic with sequenced_at update recently changed. 
can you use `if sample_obj.reads > sample_obj.application_version.application.expected_reads`

Reply:
or add a property on Sample table to check if this threshold is passed

Reply:
okej! 👍